src/replica/replica_chkpt.cpp

```diff
@@ -191,46 +198,29 @@ void replica::init_checkpoint(bool is_emergency)
         _stub->_counter_recent_trigger_emergency_checkpoint_count->increment();
 }
 
-// @ secondary
-void replica::on_copy_checkpoint(const replica_configuration &request,
```

Comment:
I notice that you update the error code of this function, will it affect previous learn process?

Reply:
it's not used in the past